ui/src/components/Tabs.vue

```diff
@@ -9,11 +9,24 @@
             :data-component="`FILENAME_PLACEHOLDER#${tab}`"
         >
             <template #label>
-                <component :is="embedActiveTab || tab.disabled || tab.locked ? 'a' : 'router-link'" @click="embeddedTabChange(tab)" :to="embedActiveTab ? undefined : to(tab)" :data-test-id="tab.name">
-                    <enterprise-tooltip :disabled="tab.locked" :term="tab.name" content="tabs">
+                <component 
+                    :is="embedActiveTab || tab.disabled || tab.locked ? 'a' : 'router-link'" 
+                    @click="embeddedTabChange(tab)" 
+                    :to="embedActiveTab ? undefined : to(tab)" 
+                    :data-test-id="tab.name"
+                >
+                    <el-tooltip v-if="tab.disabled && tab.props && tab.props.showTooltip" :content="$t('add-trigger-in-editor')" placement="top">
```

Comment:
We can not remove the `enterprise-tooltip`, as we're showing it on tabs which are enterprise only, see the last one, `Audit logs`. So get that back, and use your logic as an extra.

And also, revert all unnecessary formatting changes to keep code ownership as intact as possible.